modules/installer.nix

```diff
@@ -248,8 +248,9 @@ let cfg = config.nix-dabei; in
             serviceConfig.Type = "oneshot";
             script = ''
                 set -o errexit
-                umount --recursive /mnt
-                zpool export rpool
+                umount --verbose --recursive /mnt
+                zpool export -af
```

Comment:
Thanks! Why is `--force` needed here? I guess it should not affect any other pools unless they were mounted by the user manually? But the man page mentions that it might lead to data corruption so we should at least explain why/wheb its needed.

Was about to comment on the use of short-form flags in script, but zpool does not seem to support a long form :/

Reply:
That was me trying to be overly aggressive and overlooking the caution in the manpage. I agree with removing the `--force` flag.